libs/utils/utils.js

```diff
@@ -588,6 +588,8 @@ async function decoratePlaceholders(area, config) {
   const regex = /{{(.*?)}}/g;
   const found = regex.test(el.innerHTML);
   if (!found) return;
+  const updatedConfig = getConfig();
+  if (updatedConfig.placeholders) config.placeholders = updatedConfig.placeholders;
```

Comment:
agree with the first one.  since I need config again after 849, I want to leave it there.  But I don't see a purpose for the extra call inside the function. removed.